operators/src/source/ogr_source.rs

```diff
@@ -423,6 +426,15 @@ where
         // TODO: add OGR time filter if forced
         let dataset = Self::open_gdal_dataset(&dataset_information)?;
         let mut layer = dataset.layer_by_name(&dataset_information.layer_name)?;
+        let use_ogr_spatial_filter = dataset_information.force_ogr_spatial_filter
+            || layer.has_capability(gdal::vector::LayerCaps::OLCFastSpatialFilter);
```

Comment:
This way we use it automatically if OGR says it is possible.
So `force_ogr_spatial_filter` only has an effect if it is true and OGR says it cannot use it.
Does this make sense? I would suggest removing the parameter (in the operator spec) and just use it when OGR says it can use it.

Reply:
yes thats the way i think it is usefull. we can override the driver bbut the default is that we filter it if the driver can not provide it (fast).